### PR TITLE
CE-3321 Confirmation modal for undoing revisions

### PR DESCRIPTION
**Pull request by kamilkoterba**
https://wikia-inc.atlassian.net/browse/CE-3321

@Wikia/community-engineering

### CODE REVIEW

File: front/main/app/components/article-diff.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ export default Ember.Component.extend({
 		 */
 		undo() {
 			this.sendAction('undo');
+		},
+		showConfirmation() {
+			this.sendAction('showConfirmation');
+		},
+		closeConfirmation() {
```
Comment (Grunny) [inline front/main/app/components/article-diff.js]: This action isn't used on the article-diff component, right?

File: front/main/app/controllers/article-diff.js [hunk 2]
```diff
@@ -64,6 +67,12 @@ export default Ember.Controller.extend({
 				category: 'recent-wiki-activity',
 				label: 'undo'
 			});
-		}
+		},
+		showConfirmation() {
```
Comment (Grunny) [inline front/main/app/controllers/article-diff.js]: Put a blank line between the methods and add a JSDoc for each.

File: front/main/app/templates/components/confirmation-modal.hbs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="confirmation-message">
+    <header class="sub-head">
+        <button {{action 'close'}} class="sub-head--cancel">
+            {{svg 'back-arrow' role='img' class='icon'}}
+        </button>
+        <div class="sub-head--title"></div>
+        <button {{action 'undo'}} class="sub-head--done">{{i18n 'main.undo-label' ns='recent-wiki-activity'}}</button>
+    </header>
```
Comment (Grunny) [inline front/main/app/templates/components/confirmation-modal.hbs]: Whitespace issues.

File: front/main/app/components/confirmation-modal.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	classNames: ['confirmation-message'],
+	currentUser: Ember.inject.service(),
+	showDiffLink: false,
+	bodyText: [],
+
+	actions: {
+		/**
+		 * @returns {void}
+		 */
+		undo() {
```
Comment (lukaszkonieczny) [inline front/main/app/components/confirmation-modal.js]: I'm not sure about it but if it's common confirmation-modal component maybe we can use some more universal naming (ie. approve, confirm) for method add pass action as a parameter?

**Comment by kamilkoterba** [comment]
will be tested on feature branch

**Pull request merged**